vnext/Desktop.DLL/react-native-win32.x64.def

```diff
@@ -59,6 +59,7 @@ EXPORTS
 ??0WebSocketModule@React@Microsoft@@QEAA@XZ
 ??0NetworkingModule@React@Microsoft@@QEAA@XZ
 ?CreateAsyncStorageModule@react@facebook@@YA?AV?$unique_ptr@VCxxModule@module@xplat@facebook@@U?$default_delete@VCxxModule@module@xplat@facebook@@@std@@@std@@PEB_W@Z
+?CreateBlobModule@React@Microsoft@@YA?AV?$unique_ptr@VCxxModule@module@xplat@facebook@@U?$default_delete@VCxxModule@module@xplat@facebook@@@std@@@std@@XZ
```

Comment:
We're going to need this for URWP as well, right? Since this is a CxxModule, we can load it for both UWP and Desktop if we wanted to (see other modules loaded in OInstance). How challenging would it be to adapt other UWP modules to work with this as well so that we can share this between platforms?

Reply:
Not challenging. It will be done in a subsequent PR.

See this note from the PR description:
```
The implementation is Desktop-only.
UWP uses a different WebSocket module class.
This module will be moved into shared code once WebSocket implementations have been consolidated.
```

Because the WebSocket module is a different class (for now) in UWP, we would be doing duplicate work.
It's best to just consolidate both module in the shared base.